src/openrct2/PlatformEnvironment.cpp

```diff
@@ -53,8 +53,6 @@ class PlatformEnvironment final : public IPlatformEnvironment
         const utf8 * directoryName;
         switch (base) {
         default:
-        case DIRBASE::RCT1:
```

Comment:
This does reduce clarity. In addition, the default case should come last in the switch construct, IMO.